apps/web/src/pages/auth/QuestionnairePage.tsx

```diff
@@ -14,9 +16,9 @@ export default function QuestionnairePage() {
       ) : (
         <AuthContainer
           title="Customize your experience"
-          description="Your answers can decrease the time to get started"
+          description={IS_DOCKER_HOSTED ? 'Your answers can decrease the time to get started' : ''}
         >
-          <QuestionnaireForm />
+          {IS_DOCKER_HOSTED ? <QuestionnaireForm /> : <HubspotSignupForm />}
```

Comment:
What is the purpose of including the `AuthContainer` when there is no questionairre to fill out? What will this look like?

Reply:
Title and meta data